.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
+}
```

Comment:
This file should be deleted.

Reply:
Reminder to delete this file.

Reply:
Will deleting this be enough? How I make sure this won't happen again in future commits?